sdk/synapse/synapse-accesscontrol/README.md

```diff
@@ -0,0 +1,26 @@
+## Azure AccessControlClient SDK for JavaScript
+
+This package contains an isomorphic SDK for AccessControlClient.
+
+### Currently supported environments
+
+- Node.js version 8.x.x or higher
+- Browser JavaScript
+
+### How to Install
+
+```bash
+npm install @azure/synapse-accesscontrol
+```
+
+### How to use
+
```

Comment:
can we get some example code from the team?

Reply:
Yup, added samples in latest